driver/src/main/java/org/neo4j/driver/v1/Config.java

```diff
@@ -113,6 +119,11 @@ public TrustStrategy trustStrategy()
         return trustStrategy;
     }
 
+    public AuthToken authToken()
+    {
+        return authToken;
+    }
+
```

Comment:
Well, if you're on the JVM and you have access to this part of the object graph I don't think there's any way to keep that from happening anyway, if it's stored somewhere on the heap you can get it. We could look into techniques to obfuscate this of course.. but the JVM does not provide good help for that.

IMO this part of the API is a bit wrong - the end-user cares about the config builder, not the Config itself. I'd love to find a way to make all the methods here non-public